third_party/openthread/src/ncp/spinel.h

```diff
@@ -979,6 +979,9 @@ typedef enum
      */
     SPINEL_PROP_IPV6_ICMP_PING_OFFLOAD  = SPINEL_PROP_IPV6__BEGIN + 5, ///< [b]
 
+    SPINEL_PROP_IPV6_MULTICAST_ADDRESS_TABLE
```

Comment:
Hmm... I really need to devote a day or two to Spinel stuff this week to work out these details. I'm afraid we are coding ourselves into a corner with stuff like this.